### PR TITLE
Add lira scripts

### PR DESCRIPTION
**Pull request by samanehsan**
Add these scripts that were in the Lira repo that are related to deployment: https://github.com/HumanCellAtlas/lira/pull/170/files

### CODE REVIEW

**Review by rhiananthony — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -43,3 +43,33 @@ The gitlab.yml file defines the pipeline for the deployment. This consists of tw
 * lira deployment
 
 It is expected that unit tests and then later integration tests will be run as part of this pipeline setup.
```
Comment (rexwangcc) [inline README.md:45]: It might be more straightforward to place these instructions to a standalone readme file and have a link from the main readme? 🤔

**Review by rexwangcc — APPROVED** [review]
I left an optional comment.

File: README.md (hunk 1 above)
Comment (samanehsan) [reply README.md:45]: I think that can be a good follow-up improvement to this documentation! For now, I just copied over the README that  used to be in the Lira deploy repo

**Pull request merged**